test/SlowTests/Issues/RavenDB-16614.cs

```diff
@@ -134,13 +134,7 @@ public async Task CanHandleConflictsWithClusterTransactionIndex(Options options)
                 var arava = await session2.LoadAsync<User>("users/arava");
                 var cv = session2.Advanced.GetChangeVectorFor(arava);
                 var cti = cv.ToChangeVectorList().Where(x => x.NodeTag == ChangeVectorParser.TrxnInt).ToList();
-                if (options.DatabaseMode == RavenDatabaseMode.Single)
-                    Assert.Equal(2, cti.Count);
-                else
-                {
-                    // for sharded we have the following TRXN:12-aaa, RAFT:12-bbb | TRXN:12-aaa, RAFT:12-bbb, TRXN:44-xxx, RAFT:44-yyyy
-                    Assert.Equal(3, cti.Count);
-                }
+                Assert.Equal(2, cti.Count);
```

Comment:
here it is a conflict between 2 cluster tx, so what is the change vector in the sharded case?

Reply:
Before those PR changes:
`TRXN:12-aaa, RAFT:12-bbb, A:2:ccc | TRXN:12-aaa, RAFT:12-bbb, TRXN:44-xxx, RAFT:44-yyyy`

After the changes:
`RAFT:12-bbb, A:2:ccc | TRXN:12-aaa, RAFT:12-bbb, TRXN:44-xxx, RAFT:44-yyyy`

I don't think the `TRXN` tag should be a part of the `ChangeVector.Order`